Contest_Archive_Format.md

```diff
@@ -16,708 +17,166 @@ disk, including:
-| rgb             | string  | no       | Hexadecimal RGB value of problem color as specified in [HTML hexadecimal colors](http://en.wikipedia.org/wiki/Web_colors#Hex_triplet), e.g. '#AC00FF' or '#fff'. |
-| color           | string  | no       | Human readable color description associated to the RGB value. |
-| time_limit      | number  | no       | Time limit in seconds per test data set (i.e. per single run). |
-| test_data_count | integer | no       | Number of test data sets. |
-
-#### Differences from Contest API
-
-- The `time_limit` element is required to be a `number` that is an
-  integer multiple of `0.001` in the Contest API. 
-
-#### Examples
-
-```json
-{
-  "id": "asteroids",
-  "uuid": "cb1c4b77-b203-4943-a13f-9b89dec1ac11",
-  "label": "A",
-  "name": "Asteroid Rangers",
-  "ordinal": 1,
-  "rgb":"#00f",
-  "color": "blue",
-  "time_limit": 2,
-  "test_data_count": 10
-}
-```
-
-### Group object
-
-| Name    | Type    | Required | Description |
-| :------ | :------ | :------- | :---------- |
-| id      | ID      | yes      | Identifier of the group. |
-| icpc_id | string  | no       | External identifier from ICPC CMS. |
-| name    | string  | yes      | Name of the group. |
-| type    | string  | no       | Type of this group. |
-| hidden  | boolean | no       | If group should be hidden from scoreboard. Defaults to false if missing. |
-
-#### Differences from Contest API
-
-None.
-
-#### Examples
-
-```json
-{
-  "id": "asia",
-  "icpc_id": "7593",
-  "name": "Asia"
-}
+A package contains information regarding a single contest (corresponding to
+the `/contests/<id>/*` endpoints of the API). The API can contain information
+for several contests, to store all information a package per contest would be
+needed.
+
+Information in the API is always either in JSON format, [NDJSON]
+(contest_api#event-feed) format, or linked using a [file reference]
+(contest_api#json-attribute-types) JSON object.
+
+- The JSON returned from the endpoint `/` is stored as
+  `api.json`.
+- The JSON returned from the endpoint `/contests/<id>` is stored as
+  `contest.json`. (Notice the singular form).
+- The JSON returned from the endpoint `/contests/<id>/<endpoint>` is stored as
+  `<endpoint>.json`.
+- The NDJSON returned from the endpoint `/contests/<id>/<endpoint>` is stored as
+  `<endpoint>.ndjson`. (The only such endpoint is `event-feed`) 
+
+
+Files referenced to in `contest.json` are stored as `contest/<filename>`, and
+files referenced to in `<endpoint>.json` as `<endpoint>/<id>/<filename>`,
+where:
+- `<id>` is the ID of the endpoint object the reference is in.
+- `<filename>` is the filename specified in the file reference object.
+
+Note that the API specification requires that filenames are unique within
+endpoint objects, so this is always possible.
+
+The href in the file reference may not be valid and should be ignored, but it
+does not have to be removed.
```

Comment:
This sentence seems to jump out as needing some explanation or context. Could we explain when or why this might happen?

Also, I'm not sure what it means. Would it be valid to rewrite it as: "It is possible that the `href` in the file reference is not valid, and if it is invalid it should be ignored (but need not be removed)."

Reply:
The href might point to a location on a temporary local network, and thus not be available after the contest.

Reply:
I made an attempt of a better explanation...